lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,15 +1,7 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
 
 module SemanticPuppet
-  if defined?(GettextSetup)
-    GettextSetup.initialize(File.absolute_path('semantic_puppet/locales', File.dirname(__FILE__)))
-  end
```

Comment:
One reason to avoid patching the vendored code is that we want to remove it in [PUP-7115](https://tickets.puppetlabs.com/browse/PUP-7115).

@MosesMendoza, do you feel that we need an 1.0.1 release of semantic_puppet to resolve this (one without a hard dependency on gettext) or should it be resolved in some other way?